.travis.yml

```diff
@@ -6,8 +6,10 @@ node_js:
 env:
   - FORMDATA_VERSION=1.0.0
   - FORMDATA_VERSION=2.1.0
+  - ENCODING=yes
 before_script:
   - 'if [ "$FORMDATA_VERSION" ]; then npm install form-data@^$FORMDATA_VERSION; fi'
+  - 'if [ "$ENCODING" = "yes" ]; then npm install encoding; fi'
```

Comment:
You could do

```yaml
env:
  - FORMDATA_VERSION=1.0.0
  - FORMDATA_VERSION=1.0.0 ENCODING=yes
  - FORMDATA_VERSION=2.1.0
  - FORMDATA_VERSION=2.1.0 ENCODING=yes
```

so that the entire matrix will run. See https://docs.travis-ci.com/user/environment-variables/.